python/sklearnserver/setup.py

```diff
@@ -21,7 +21,7 @@
 ]
 setup(
     name='sklearnserver',
-    version='0.1.0',
+    version='0.2.1',
     author_email='singhan@us.ibm.com',
```

Comment:
so we need to get every framework in e2e if not there? issues like this can be caught then?